operantanalysis/operantanalysis.py

```diff
@@ -95,7 +93,16 @@ def respondingduringcueanditi(timecode, eventcode, codeon, codeoff):
                 itipoke += 1
         itipokerpm = itipoke / (cuelengthsec / 60)
         allpokeitirpm += [itipokerpm]
-    print(allpokeitirpm)
-    print(allpokerpm)
     
     return round(statistics.mean(allpokerpm), 3), round(statistics.mean(allpokeitirpm), 3)
+
+
+def leverpressing(eventcode, lever1, lever2=False):
+    lever1presses = eventcode.count(lever1)
+    if bool(lever2):
```

Comment:
no need for bool() here:
```suggestion
    if lever2:
```

Reply:
Fixed in new branch